src/components/CommentContainer/CommentContainer.tsx

```diff
@@ -140,10 +142,10 @@ export const CommentContainer = ({
 				onRecommend={onRecommend}
 			/>
 
-			<>
+			<React.Fragment>
```

Comment:
I'm not following along but is this now how we need to do these fragments now? That would be a shame.